<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotkeyWidget</class>
 <widget class="QWidget" name="HotkeyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBoxCtrl">
     <property name="text">
      <string>Ctrl</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxShift">
     <property name="text">
      <string>Shift</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAlt">
     <property name="text">
      <string>Alt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxWin">
     <property name="text">
      <string>Win</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxKeys">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxCtrl</tabstop>
  <tabstop>checkBoxShift</tabstop>
  <tabstop>checkBoxAlt</tabstop>
  <tabstop>checkBoxWin</tabstop>
  <tabstop>comboBoxKeys</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
